src/vend-number.js

```diff
@@ -60,16 +81,17 @@ VendNumber.vn = function (value) {
  * @param [decimalPoints=2]
  *        The number of decimal points to round the passed value to, or two.
  *
+ * @param [roundingMode=BigNumber.ROUND_HALF_UP]
+ *        The required rounding mode. Defaults to ROUND_HALF_UP (4).
```

Comment:
That's the value of `ROUND_HALF_UP`; see [here]( https://github.com/MikeMcl/bignumber.js/blob/v2.3.0/bignumber.js#L76).